atlasdb-timelock-server/src/test/java/com/palantir/atlasdb/timelock/TimeLockServerTest.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2016 Palantir Technologies
+ * Copyright 2017 Palantir Technologies
```

Comment:
Ah, this is an artifact of what I did:

1. refactored `TimeLockServerTest` to `ServerImplementationTest`
2. created new `TimeLockServerTest` (which now says `2017`, though as far as Git is concerned it's not a new file).

I've changed it back to read `2016`.